feat(experiments): add documentation for retention metric. (#14135)
PostHog/posthog.com: contents/docs/experiments/metrics.mdx

Both the numerator and denominator support all aggregation methods (count, sum, average, unique values). You can even use the same event with different properties for each part of the ratio, such as dividing total revenue by total quantity sold from purchase events.

### Retention

Use retention metrics to measure if users come back to perform a specific action within a defined time window after an initial action. This helps you measure engagement patterns, feature stickiness, and long-term user behavior.

Retention metrics track two events and a time window:

| Event | Description | Examples |
|-------|------------|----------|
| Start event | The initial action that begins tracking | Signup, complete onboarding, first purchase |

> | Completion event | The action you want users to return for | Login, use feature, make another purchase |

Reviewer: ```suggestion
| Completion event | The action you want users to return for | Login, feature use, make another purchase |
```

> | Retention window | When the completion event should occur | Between 1 and 7 days after start event |

Author: it's a suggestion. Just and example.

Both the numerator and denominator support all aggregation methods (count, sum, average, unique values). You can even use the same event with different properties for each part of the ratio, such as dividing total revenue by total quantity sold from purchase events.

### Retention

> Use retention metrics to measure if users come back to perform a specific action within a defined time window after an initial action. This is ideal for measuring engagement patterns, feature stickiness, and long-term user behavior.

Reviewer 2: ```suggestion
Use retention metrics to measure if users come back to perform a specific action within a defined time window after an initial action. This helps you measure engagement patterns, feature stickiness, and long-term user behavior.
```
